doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,185 @@
+    - Advocating for, or encouraging, any of the above behaviour.
+
+5. Be concise. Keep in mind that what you write once will be read by hundreds
+   of persons. Writing a short email means people can understand the
+   conversation as efficiently as possible. Short emails should always strive
+   to be empathetic, welcoming, friendly and patient. When a long explanation
+   is necessary, consider adding a summary.
+
+6. Try to bring new ideas to a conversation so that each mail adds something
+   unique to the thread, keeping in mind that the rest of the thread still
+   contains the other messages with arguments that have already been made.
+
+7. Try to stay on topic, especially in discussions that are already fairly
+   large.
+
+8. Step down considerately. Members of every project come and go. When somebody
+   leaves or disengages from the project they should tell people they are
+   leaving and take the proper steps to ensure that others can pick up where
+   they left off. In doing so, they should remain respectful of those who
+   continue to participate in the project and should not misrepresent the
+   project's goals or achievements. Likewise, community members should respect
+   any individual's choice to leave the project.
+
+
+Diversity Statement
+-------------------
+
+The SciPy project welcomes and encourages participation by everyone. We are
+committed to being a community that everyone feels good about joining. Although
+we may not be able to satisfy everyone, we will always work to treat everyone
+well.
+
+No matter how you identify yourself or how others perceive you: we welcome you.
+Though no list can hope to be comprehensive, we explicitly honour diversity in:
+age, culture, ethnicity, genotype, gender identity or expression, language,
+national origin, neurotype, phenotype, political beliefs, profession, race,
+religion, sexual orientation, socioeconomic status, subculture and technical
+ability.
+
+Though we welcome people fluent in all languages, SciPy development is
+conducted in English.
+
+Standards for behaviour in the SciPy community are detailed in the Code of
+Conduct above. We expect participants in our community to meet these standards
+in all their interactions and to help others to do so as well.
+
+
+Reporting Guidelines
+--------------------
+
+While this code of conduct should be adhered to by participants, we recognize
+that sometimes people may have a bad day, or be unaware of some of the
+guidelines in this code of conduct. When that happens, you may reply to them
+and point out this code of conduct. Such messages may be in public or in
+private, whatever is most appropriate. However, regardless of whether the
+message is public or not, it should still adhere to the relevant parts of this
+code of conduct; in particular, it should not be abusive or disrespectful.
+
+If you believe someone is violating this code of conduct, you may reply to them
```

Comment:
First sentence repeats part of the previous paragraph.

Reply:
fixed